{{cookiecutter.project_slug}}/templates/500_multilingual.html

```diff
@@ -0,0 +1,39 @@
+{% load i18n %}
```

Comment:
Could we drop this file and stick with the regular 500?

I don't think there's a need to translate it, and chances are the Sentry reporting form will be English anyway.